docs/concepts/runtime_capabilities.md

```diff
@@ -103,21 +103,59 @@ Both the [`elastic` connector](../reference/connectors/elastic.md) and the [`htt
 
 ### Garuanteed Delivery
 
-Tremor uses the contraflow mechanism to implement **Event delivery acknowledgements**. In a nutshell it works like this:
+Tremor uses the contraflow mechanism to implement **Event delivery acknowledgments**. In a nutshell, it works like this:
 
-* When a connector receives an event and handles it successfully without error (e.g. sends it off via TCP) and event delivery acknowledgement is sent backwards via contraflow, if required, which is flagged in each event, usually by the originating connector.
-* When a connector fails to handle an event a delivery failure message is sent backwards via contraflow, if required.
+* When a connector receives an event and handles it successfully without error (e.g., sends it off via TCP) and event delivery acknowledgment is sent backward via contraflow, if required, which is flagged in each event, usually by the originating connector.
```

Comment:
Pls change `acknowledgment` back to `acknowledgement` everywhere. Oxford says the `e` belongs there.

```suggestion
* When a connector receives an event and handles it successfully without error (e.g., sends it off via TCP) and event delivery acknowledgement is sent backward via contraflow, if required, which is flagged in each event, usually by the originating connector.
```